riscv/src/register/pmpcfgx.rs

```diff
@@ -76,6 +118,37 @@ impl Pmpcsr {
             locked: (byte & (1 << 7)) != 0,
         }
     }
+
+    /// Attempts to take the register contents, and translate into a Pmp configuration struct.
+    #[inline]
+    pub fn try_into_config(&self, index: usize) -> Result<Pmp> {
+        let max = if cfg!(riscv32) {
+            Ok(4usize)
+        } else if cfg!(riscv64) {
+            Ok(8usize)
+        } else {
+            Err(Error::Unimplemented)
+        }?;
```

Comment:
```suggestion
        let max = if cfg!(riscv32) {
            4usize
        } else if cfg!(riscv64) {
            8usize
        } else {
            return Err(Error::Unimplemented);
        };
```

Reply:
Is there a reason you prefer the explicit `return` keyword?

From what I've seen in style guides, and personal practice, having one `return` statement max is preferred. Every early return should utilize the `?` operator where possible.